Makefile

```diff
@@ -33,6 +33,8 @@ else
 	@echo "Not installing golint, since we don't expect to lint on" $(GO_VERSION)
 endif
 
+VET_SKIP_ERRORF=grep -v -e "possible formatting directive in Error call"
```

Comment:
Is this a pervasive issue in zap, or is there one particular place where vet complains?